erts/doc/src/erl_dist_protocol.xml

```diff
@@ -1096,6 +1096,15 @@ DiB == gen_digest(ChA, ICA)?
           <p>The node supports the new connection setup handshake (version 6)
 	  introduced in OTP 23.</p>
         </item>
+        <tag><marker id="DFLAG_UNLINK_ID"/><c>-define(DFLAG_UNLINK_ID, 16#2000000).</c></tag>
+        <item>
+	  <p>Use the <seeguide marker="#new_link_protocol">new link protocol</seeguide>.</p>
+	  <note><p>This flag will become mandatory in OTP 26.</p></note>
+	  <p>When this flag is set, the link protocol previously used is
+	  replaced by a new link protocol. This since the previously used
+	  protocol could cause process links to end up in an inconsistent
```

Comment:
```suggestion
	  <p>When this flag is set, the previously used link protocol is
	  replaced by a new link protocol. The former
	  could cause process links to end up in an inconsistent
```

Reply:
I still don't think it properly conveys the change; I just tried to clear it up.

Reply:
I've added an explanation of what the distribution flags/capability flags are in the beginning of that section and changed the documentation of `DFLAG_UNLINK_ID`. Hope that clears it up.